client/src/admin/AdminHeader.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+const AdminHeader = () => (
+  <div>
+    <h1 className="text--center text__margin--70">Submit New Notification</h1>
+    <div className="container--blue">
+      On this page you can set up an alert for the following services
+      provided by Interior Geospatial Emergency Management System (IGEMS).
+      Subscribers will receive an alert via text or email.
```

Comment:
"Subscribers in the selected area" I think. We need to make it explicit that they are supposed to click on a zone.